core/utility/constants.py

```diff
@@ -23,7 +23,14 @@
 CONSTANTS_JS_FILEPATH = os.path.join('static', 'assets', 'constants.js')
 
 def parse_json_from_js(js_file):
-    """Extracts JSON object from JS file."""
+    """Extracts JSON object from JS file.
+    Args:
+        js_file: file. A stream handling object of constants.js.
+
+    Returns:
+        dict(str, str). The dictionary containing variable name as key and
```

Comment:
Why do you need the last part (unicode strings)? I thought we could represent numbers, booleans, even null in this JSON representation.

(If that's the case, then something like "A dict mapping constant names to their values." might be better.)

Reply:
So I was getting `u'abcdef'` when I print out the dict values. Anyhow, I re-word the docstring.